tex/chProofs.tex

```diff
@@ -40,9 +41,9 @@
 to prove theorems, focusing on simple statements and basic proof
 commands. In the course of this book, we will see that choosing the
 right way to state a proposition formally can be a rather delicate
-matter. For equivalent wordings of a same propositions, some can be
+matter. For equivalent wordings of a same proposition, some can be
```

Comment:
While we are near "a same": according to the Longman DOCE dictionary (5th Ed.): "Same never has 'a' before it". There are 4 occurrences of "a same" in the book: chProgramming.tex (1), chProofs.tex (1), chSpecification.tex (2).